packages/ember-glimmer/tests/integration/syntax/each-in-test.js

```diff
@@ -270,7 +279,6 @@ moduleFor('Syntax test: {{#each-in}}', class extends EachInTest {
         'Smartphones': 8203,
         'JavaScript Frameworks': Infinity
       });
-      this.rerender();
```

Comment:
This `rerender` is not necessary. Both engines correctly "observe" the whole object changing.